vignettes/TADAGeospatialFunctions.Rmd

```diff
@@ -0,0 +1,236 @@
+all EPA Region 5 states.
+
+**Note: TADA is still under development. New functionality is added weekly, and sometimes we need to make bug fixes in response to tester and user feedback. We appreciate your feedback, patience, and interest in these helpful tools.**
+
+**If you are interested in contributing to TADA development, more information is available at [Contributing] (https://usepa.github.io/TADA/articles/CONTRIBUTING.html). We welcome collaboration with external partners.**
+
+## Install and load packages
+
+First, install and load the remotes package specifying the repo. This
+is needed before installing TADA because it is only available on GitHub.
+
+```{r install_remotes, eval = F, results = 'hide', message = FALSE, warning = FALSE}
+install.packages("remotes",
+  repos = "http://cran.us.r-project.org"
+)
+library(remotes)
+```
+
+Next, install and load TADA using the remotes package. TADA R Package dependencies will also be downloaded automatically from CRAN
+with the TADA install. You may be prompted in the console to update dependency packages that have more recent versions available. If you see this prompt, it is recommended to update all of them (enter 1 into the console).
+
+```{r install_TADA, eval = F, results = 'hide', message = FALSE, warning = FALSE}
+remotes::install_github("USEPA/TADA",
+  ref = "R5_package_demo",
+  dependencies = TRUE
+)
+```
+
+Finally, use the **library()** function to load the TADA R Package into your R session.
+
+```{r library, results = 'hide', message = FALSE, warning = FALSE}
+library(TADA)
+```
+
+## Help pages
+
+All TADA R package functions have their own individual help pages, listed on the [Function reference](https://usepa.github.io/TADA/reference/index.html) page on the GitHub site. Users can also access the help page for a given function in R or RStudio using the following format (example below): `?TADA::[name of TADA function]`
+
+```{r help_pages, message = FALSE}
+# Access help page for TADA_DataRetrieval
+?TADA_DataRetrieval
+```
+
+## Geospatial Functions in TADA
+
+This vignette represents three functions aimed towards completing the TADA
+Module 2 goal: *Match WQP sites with ATTAINS assessment units. Assist user
+with reviewing sites and Assessment Units (AUs) on a map and deciding to keep
+or exclude sites (systematically first, maybe using catchments as a buffer
+around the AU points/line/polygons for consistency, and then let users make
+manual edits to include or exclude sites (main requirement)). Consider adding
+option to create new AUs for new areas where WQP sites are available but no AU
+exists yet (secondary requirement). Output is a simple spreadsheet linking
+ATTAINS AU IDs with WQP Location IDs.*
```

Comment:
The audience for this paragraph seems different to me than the other contents of the vignette. If the idea is for this to be a vignette for end users then I think this can get dropped and instead the info could just be referenced in the narrative of the upcoming PR to the main package

Reply:
agreed. will copy and paste this into the PR and modify what goes in vignette.